configuration/external-server/config.json

```diff
@@ -1,27 +1,29 @@
 {
-    "company_name": "bringauto",
-    "car_name": "virtual_vehicle",
-    "mqtt_timeout": 60,
-    "timeout": 60,
-    "send_invalid_command": false,
-    "sleep_duration_after_connection_refused": 0.5,
-    "modules": {
-      "1": {
-        "lib_path": "/home/bringauto/modules/libmission_external_server.so",
-        "config": {
-            "api_url": "http://http-api:8080",
-            "api_key": "StaticAccessKeyToBeUsedByDevelopersOnEtna"
-        }
-      },
-      "2": {
-        "lib_path": "/home/bringauto/modules/libio_external_server.so",
-        "config": {
-        }
-      },
-      "1000": {
-        "lib_path": "/home/bringauto/modules/libexample_external_server.so",
-        "config": {
-        }
+  "company_name": "bringauto",
+  "car_name": "virtual_vehicle",
+  "mqtt_address": "mosquitto",
+  "mqtt_port": 1883,
+  "mqtt_timeout": 60,
+  "timeout": 60,
+  "send_invalid_command": false,
+  "sleep_duration_after_connection_refused": 0.5,
+  "log_files_directory": "/home/bringauto/log",
+  "log_files_to_keep": 5,
+  "log_file_max_size_bytes": 50000000,
+  "modules": {
+    "1": {
+      "lib_path": "/home/bringauto/modules/libmission_external_server.so",
+      "config": {
+          "api_url": "http://http-api:8080/v2/protocol",
+          "api_key": "StaticAccessKeyToBeUsedByDevelopersOnEtna"
+      }
+    },
+    "2": {
+      "lib_path": "/home/bringauto/modules/libio_external_server.so",
+      "config": {
+        "api_url": "http://http-api:8080/v2/protocol",
+        "api_key": "StaticAccessKeyToBeUsedByDevelopersOnEtna"
       }
     }
```

Comment:
please add request rate limit config to modules:

"maxRequestsThresholdCount": 5,
"maxRequestsThresholdPeriodMs": 1000,
"delayAfterThresholdReachedMs": 500,
"retryRequestsDelayMs": 220